### PR TITLE
Update LGPL licenses

### PR DESCRIPTION
**Pull request by PuneetGopinath**
Request to add a new Trove classifier.

## The name of the classifier(s) you would like to add:

* `License :: OSI Approved :: GNU Lesser General Public License v2.0 only (LGPLv2.0-only)`
* `License :: OSI Approved :: GNU Lesser General Public License v2.0 or later (LGPLv2.0-or-later)`
* `License :: OSI Approved :: GNU Lesser General Public License v2.1 only (LGPLv2.1-only)`
* `License :: OSI Approved :: GNU Lesser General Public License v2.1 or later (LGPLv2.1-or-later)`

## Why do you want to add this classifier?
See #68, I wanted to add `LGPL v2.1` but after seeing #17, I added all revisions required for Lesser General Public License (in different versions) and depreciated the old <del>ones</del> classifiers.

### CODE REVIEW

**Comment by remram44** [comment]
If you are putting short IDs in parentheses, I think you should consider using [the official SPDX identifiers](https://spdx.dev/licenses/) of those licenses:


* `License :: OSI Approved :: GNU Lesser General Public License v2.0 (LGPL-2.0-only)`
* `License :: OSI Approved :: GNU Lesser General Public License v2.0 or later (LGPL-2.0-or-later)`
* `License :: OSI Approved :: GNU Lesser General Public License v2.1 (LGPL-2.1-only)`
* `License :: OSI Approved :: GNU Lesser General Public License v2.1 or later (LGPL-2.1-or-later)`
* `License :: OSI Approved :: GNU Lesser General Public License v3.0 (LGPL-3.0-only)`
* `License :: OSI Approved :: GNU Lesser General Public License v3.0 or later (LGPL-3.0-or-later)`

**Comment by PuneetGopinath** [comment]
@remram44 I updated

**Comment by PuneetGopinath** [comment]
Sorry now it's night at India, will talk with you tomorrow

**Comment by PuneetGopinath** [comment]
@remram44 So should I keep the new ones and the old ones without depreciating the old ones.
@di Mentioned (in https://github.com/pypa/trove-classifiers/issues/17#issuecomment-385027197) that LGPL license used by < 831 projects
| LICENSE | no of projects |
| - | - |
| GNU Lesser General Public License v2 (LGPLv2) | 128 |
| GNU Lesser General Public License v2 or later (LGPLv2+) | 175 |
| GNU Lesser General Public License v3 (LGPLv3) | 831 |
| GNU Lesser General Public License v3 or later (LGPLv3+) | 502 |

So at least can we add LGPL v2.0 and v2.1, they are used by < 175 projects

**Comment by remram44** [comment]
I don't know :neutral_face: I guess we need to wait for the higher-ups to get to a consensus...

**Comment by PuneetGopinath** [comment]
What do you say? @di

**Comment by PuneetGopinath** [comment]
@di No progress?? Or are you busy?

**Comment by PuneetGopinath** [comment]
@remram44 Are there any other collaborators for this repo, @di Is not replying
If yes, ask those other collaborators to review my pr.

**Comment by pradyunsg** [comment]
Hi there!

This project is maintained by volunteers, who have limited availability. They will come around to this as they are able to find the free time and motivation to do so. @-mentioning folks does not serve to speed up anything, beyond causing annoying notifications for the folks being mentioned.

I'd suggest being patient here and following up about a week from now, if there's no response.

**Comment by PuneetGopinath** [comment]
> Hi there!
> 
> This project is maintained by volunteers, who have limited availability. They will come around to this as they are able to find the free time and motivation to do so. @-mentioning folks does not serve to speed up anything, beyond causing annoying notifications for the folks being mentioned.
> 
> I'd suggest being patient here and following up about a week from now, if there's no response.

Thanks

**Comment by PuneetGopinath** [comment]
A week over

**Comment by PuneetGopinath** [comment]
Is this going to be discussed

**Comment by PuneetGopinath** [comment]
2 months over

**Comment by di** [comment]
I think this is blocked on making a decision on #17. In the meantime, projects can always use the [`License`](https://packaging.python.org/specifications/core-metadata/#license) field to declare a license if there is not a sufficient classifier.

File: src/trove_classifiers/__init__.py [hunk 1]
```diff
@@ -768,4 +770,8 @@
 deprecated_classifiers = {
     "Natural Language :: Ukranian": ["Natural Language :: Ukrainian"],
     "Topic :: Communications :: Chat :: AOL Instant Messenger": [],
+    "License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)",
+    "License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
```
Comment (di) [inline src/trove_classifiers/__init__.py:776]: This is a dict, these should be a mapping from a deprecated classifier name to a list of one or more classifiers that should be used in place of it.

Comment (PuneetGopinath) [reply src/trove_classifiers/__init__.py:776]: Okay, should I update. Anyways you told this is blocked.

**Comment by Alireza1369newi** [comment]
gh pr checkout 69

File: src/trove_classifiers/__init__.py [hunk 2]
```diff
@@ -243,10 +243,12 @@
     "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
     "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
     "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
-    "License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)",
-    "License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)",
-    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
-    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "License :: OSI Approved :: GNU Lesser General Public License v2.0 only (LGPLv2.0-only)",
```
Comment (Alireza1369newi) [inline src/trove_classifiers/__init__.py:246]: **
**

**Review by Alireza1369newi — COMMENTED** [review]
Pip

File: src/trove_classifiers/__init__.py (hunk 2 above)
Comment (Alireza1369newi) [reply src/trove_classifiers/__init__.py:246]: > **
> **

**Pull request closed without merging**